File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;


import android.os.Bundle;

import android.content.Intent;
import android.os.Handler;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
        @Override
        protected void onCreate(Bundle savedInstanceState) {
            super.onCreate(savedInstanceState);
            setContentView(R.layout.activity_main);

            // Tempo de espera da tela de splash (3 segundos)
            new Handler().postDelayed(new Runnable() {
                @Override
                public void run() {
                    // Inicia a próxima Activity (por exemplo, LoginActivity)
                    Intent intent = new Intent(MainActivity.this, TelaActivity.class);
                    startActivity(intent);
                    finish(); // Fecha a MainActivity
                }
            }, 3000); // 3000 milissegundos = 3 segundos
        }
    }

File: TesteBD2/app/src/main/java/com/example/testebd/MainActivity.kt
package com.example.testebd

import android.os.Bundle
import android.widget.ArrayAdapter
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.testebd.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(binding.root)
            binding=ActivityMainBinding.inflate(layoutInflater)

        //Executar comandos sql
        val db = Conexao(this)
        //Armazenar os dados retornado pelo select, que está em listaDados
        val listaDados = db.buscaDados()
        //Metodo paa exibir os dados na tela layout
        binding.listView.adapter = ArrayAdapter(this,
            android.R.layout.simple_list_item_1, listaDados)




        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
    }
}
